src/notation/view/pianokeyboard/pianokeyboardview.cpp

```diff
@@ -267,6 +267,7 @@ void PianoKeyboardView::paintWhiteKeys(QPainter* painter, const QRectF& viewport
             int octaveNumber = (key / 12) - 1;
 
             QString octaveLabel = "C" + QString::number(octaveNumber);
+            octaveLabel = muse::qtrc("global/pitchName", octaveLabel.toStdString().c_str());
```

Comment:
I believe this is unsafe, because the `std::string` is destructed immediately after `c_str()` has been called, meaning that the pointer passed to `qtrc` is dangling. This would be solved by making `octaveLabel` immediately a std::string instead of QString, because then the lifetime of the std::string and its data is until the end of the scope, which is enough here.

Reply:
I don't think that would solve anything, would it? (and `octaveNumber` isn't static either)

Reply:
Probably, but how is this here different from 

https://github.com/musescore/MuseScore/blob/790e7493e5c0c7825f49b07ed4a8299229e60dfa/src/notation/internal/instrumentsrepository.cpp#L226

Or is that simply having the same issue?

Reply:
> MuseScore/src/notation/internal/instrumentsrepository.cpp

I saw that as well, but I think @cbjeukendrup is right, seems to be a dangling pointer issue. Will fix accordingly. Thanks!